ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -146,6 +148,7 @@
   :ssl?                 - allow connections over HTTPS
   :ssl-port             - the SSL port to listen on (defaults to 443, implies
                           :ssl? is true)
+  :ssl-context          - the SSLContext to use for SSL connections
```

Comment:
As it's optional, this should be changed to:

```
  :ssl-context          - an optional SSLContext to use for SSL connections
```

Reply:
I've updated the description, thanks.